Scripts/lint.sh

```
@@ -1,45 +1,93 @@
 	exit
-elif [ "$LINT_MODE" == "STRICT" ]; then
-	SWIFTFORMAT_OPTIONS=""
+elif [ "$LINT_MODE" = "STRICT" ]; then
+	SWIFTFORMAT_OPTIONS="--strict --configuration .swift-format"
 	SWIFTLINT_OPTIONS="--strict"
+	STRINGSLINT_OPTIONS="--config .strict.stringslint.yml"
 else 
-	SWIFTFORMAT_OPTIONS=""
+	SWIFTFORMAT_OPTIONS="--configuration .swift-format"
 	SWIFTLINT_OPTIONS=""
+	STRINGSLINT_OPTIONS="--config .stringslint.yml"
```

Shellcheck (0.11.0)

[warning] STRINGSLINT_OPTIONS appears unused. Verify use (or export if used externally).

(SC2034)